### PR TITLE
fix: remove georli & drop emoji support for now

### PR DESCRIPTION
**Pull request by hamdiallam**
Validation failures
- Goerli Network Shutdown
- LikeButton.eth token entry. Override the heart emoji symbol to LIKE until we can fix validation to support emojis

### CODE REVIEW

**Comment by hamdiallam** [comment]
* **#694** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ethereum-optimism.github.io/694?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @hamdiallam and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: data/APT/data.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Aptos",
+  "name": "Aptos Coin",
```
Comment (rcbgr) [inline data/APT/data.json:2]: Is this correct? i thought this was just Aptos now

File: data/LIKE/data.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (rcbgr) [inline data/LIKE/data.json:1]: This is a new addition?

File: data/INSURE/data.json [hunk 3]
```diff
@@ -1,19 +0,0 @@
-{
```
Comment (rcbgr) [inline data/INSURE/data.json:1]: Why are we deleting the entire token?

**Review by rcbgr — CHANGES_REQUESTED** [review]
Overall, LGTM. There appears to be a new token in there. Was this properly reviewed? There were also some removes that seemed opinionated.

File: data/APT/data.json (hunk 1 above)
Comment (hamdiallam) [reply data/APT/data.json:2]: I'll have to add an override. The top level name just needs to match the name of the L1 token. The override will allow us to display as "Aptos" though

File: data/LIKE/data.json (hunk 2 above)
Comment (hamdiallam) [reply data/LIKE/data.json:1]: This was renamed from the "❤️" directly which was causing problems with validation

File: data/INSURE/data.json (hunk 3 above)
Comment (hamdiallam) [reply data/INSURE/data.json:1]: The whole token was a goerli-only token

**Review by roninjin10 — CHANGES_REQUESTED** [review]
Because you removed a token you will need to do a major version bump

**Review by roninjin10 — APPROVED** [review]
npm run test -u should fix the test failure

**Review by roninjin10 — APPROVED** [review]

**Pull request merged**